### PR TITLE
Connor fixes

### PR DESCRIPTION
**Pull request by ConnorOgilvie91**

### CODE REVIEW

File: BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift [hunk 1]
```diff
@@ -230,6 +232,24 @@ class AuthenticatedHealthRecordsAPIWorker: NSObject {
             }
         }
     }
+    // Temporary workaround for dumb queueit issues on prod
+    private func getQueueItWorkingByHittingTestResultsEndpoint(authCredentials: AuthenticationRequestObject) {
```
Comment (AmirShayegh) [inline BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift:236]: nice name lol

File: BCVaccineCard/BCVaccineCard/Helpers/Extensions/UIView+Ext.swift [hunk 2]
```diff
@@ -72,8 +72,10 @@ extension UIView {
     }
     
     func endLoadingIndicator() {
-        if let existing = self.viewWithTag(Constants.UI.LoadingIndicator.backdropTag) {
-            existing.removeFromSuperview()
+        DispatchQueue.main.async {
```
Comment (AmirShayegh) [inline BCVaccineCard/BCVaccineCard/Helpers/Extensions/UIView+Ext.swift:75]: right

**Review by AmirShayegh — APPROVED** [review]

File: BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift (hunk 1 above)
Comment (ConnorOgilvie91) [reply BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift:237]: yup haha -  figured we're removing it as soon as queue it is removed from the HG side, so

File: BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift [hunk 3]
```diff
@@ -214,12 +214,14 @@ class AuthenticatedHealthRecordsAPIWorker: NSObject {
             }
         }
         guard let types = specificFetchTypes else {
+            self.getQueueItWorkingByHittingTestResultsEndpoint(authCredentials: authCredentials)
             self.getAuthenticatedVaccineCard(authCredentials: authCredentials)
-            self.getAuthenticatedTestResults(authCredentials: authCredentials)
             self.getAuthenticatedMedicationStatement(authCredentials: authCredentials, protectiveWord: protectiveWord ?? authManager.protectiveWord, initialProtectedMedFetch: initialProtectedMedFetch)
             self.getAuthenticatedLaboratoryOrders(authCredentials: authCredentials)
+            self.getAuthenticatedTestResults(authCredentials: authCredentials)
```
[the remark this answers is not in the document]
Comment (ConnorOgilvie91) [reply BCVaccineCard/BCVaccineCard/Helpers/APIWorkers/AuthenticatedHealthRecordsAPIWorker.swift:222]: The first is a dummy call, that hits the test endpoint, but we don't handle it in core data or anything - it's just used to get queue it setup properly. The first endpoint we call never works, and the retry doesn't happen properly due to an incorrect status code, so this is the workaround

**Pull request merged**